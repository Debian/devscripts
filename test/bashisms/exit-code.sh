#!/bin/sh

# http://bugs.debian.org/687450
exit -- 2 # BASHISM
exit 255
exit 256 # BASHISM
exit -1 # BASHISM
