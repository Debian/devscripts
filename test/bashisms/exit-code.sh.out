possible bashism in bashisms/exit-code.sh line 4 ('exit --' should be 'exit' (idem for return)):
exit -- 2 # BASHISM
possible bashism in bashisms/exit-code.sh line 6 (exit|return status code greater than 255):
exit 256 # BASHISM
possible bashism in bashisms/exit-code.sh line 7 (exit|return with negative status code):
exit -1 # BASHISM
